torch/_torch_docs.py

```diff
@@ -6672,18 +6672,30 @@ def merge_dicts(*dicts):
 If :attr:`some` is ``True``, then this function returns the thin (reduced) QR factorization.
 Otherwise, if :attr:`some` is ``False``, this function returns the complete QR factorization.
 
+.. warning:: ``torch.qr`` is deprecated. Please use ``torch.linalg.`` :meth:`~torch.linalg.qr`
+             instead, which provides a better compatibility with
```

Comment:
I think we can remove everything after "instead," and let users discover what linalg.qr does on its page.

Reply:
785136fb37